spectacles/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.8-alpine
+
+ENV version=19.10b0
```

Comment:
This version doesn't seem to be from `spectacles`, the current one is `v0.1.1`
https://github.com/spectacles-ci/spectacles/releases

Reply:
```suggestion
ENV version=0.1.1
```